hr_evaluation_360/models/hr_evaluation_plan_phase.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Luis Felipe Mileo - <mileo@kmee.com.br> - KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from dateutil import parser
+import time
+
+from openerp import api, fields, models
+
+evaluation_360_list = []
+
+
+def evaluation_360(evaluation_func):
+    evaluation_360_list.append(evaluation_func)
+    return evaluation_360
+
+
+class HrEvaluationPlanPhase(models.Model):
+
+    _inherit = 'hr_evaluation.plan.phase'
+
+    action = fields.Selection(
+        selection_add=[
+            ('360', u'360-degree feedback'),
+            ('360-anonymous', u'360-degree anonymous feedback'),
+        ]
+    )
+
+
+class HrEvaluationEvaluation(models.Model):
+
+    _inherit = 'hr_evaluation.evaluation'
+
+    @evaluation_360
+    def _get_360_evaluation_child(self, evaluation):
+        return evaluation.employee_id.child_ids
+
+    @evaluation_360
+    def _get_360_evaluation_parent(self, evaluation):
+        return evaluation.employee_id.parent_id
+
+    @evaluation_360
+    def _get_360_evaluation_myself(self, evaluation):
+        return evaluation.employee_id
+
+    @evaluation_360
+    def _get_360_evaluation_department(self, evaluation):
+        return self.env['hr.employee'].search(
+            [('department_id', '=',
+              evaluation.employee_id.department_id.id)]
+        )
+
+    @api.multi
+    def button_plan_in_progress(self):
```

Comment:
Is this overriding an `hr_evaluation` method? We should perform a `super()` call somewhere in order not to break the inheritance chain.
Also, the logic added is non-trivial enough to deserve a docstring explanation .